contracts/epochs/FlowDKG.cdc

```diff
@@ -331,6 +346,35 @@ pub contract FlowDKG {
         return self.uniqueFinalSubmissions
     }
 
+    /// Gets the native threshold that the submission count needs to exceed to be considered complete
+    pub fun getNativeSuccessThreshold(): UInt64 {
+        return UInt64((self.getConsensusNodeIDs().length-1)/2)
+    }
+
+    /// Gets the safe threshold that the submission count needs to exceed to be considered complete
+    /// (always greater than or equal to the native success threshold)
+    pub fun getSafeSuccessThreshold(): UInt64 {
+        var threshold = self.getNativeSuccessThreshold()
+
+        // Get the safety rate percentage
+        if let safetyRate = self.getSafeThresholdPercentage() {
+
+            let safeThreshold = UInt64(safetyRate * UFix64(self.getConsensusNodeIDs().length))
```

Comment:
This is fine but it's slightly different than what I wrote in the issue (the contract requires the floor of (rate*nodes) + 1  instead of just the floor). 
It's fine, but we should keep this into account when debugging/testing.